sapan-parikh/SAPAN_PARIKH.md

```diff
@@ -0,0 +1,16 @@
+# Not All Tests are the Same
+
+I have gotten an opportunity to work on projects where I learned how automated testing is and that not all automated tests are the same. They differ from each other in at least two ways,speed of execution and risk of failure.
```

Comment:
Suggestion: I have worked on projects where I learned how automated tests are not all the same..